avm/res/app/managed-environment/main.bicep

```
@@ -319,4 +342,4 @@ type storageType = {
 
   @description('Required. File share name.')
   shareName: string
-}
+}?
```

```suggestion
}
```
Parameter is already (correctly) nullable